airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -122,13 +122,7 @@ def _create_pod_id(
     """
     if not (dag_id or task_id):
         raise ValueError("Must supply either dag_id or task_id.")
-    name = ""
-    if dag_id:
-        name += dag_id
-    if task_id:
-        if name:
-            name += "-"
-        name += task_id
+    name = f"{dag_id or ''}-{task_id or ''}"
```

Comment:
Strictly speaking this is not a refactor as the `-` is always included in the new version, but not the old version.

Reply:
Thanks, fixed.